prime-router/src/main/kotlin/metadata/Mappers.kt

```diff
@@ -529,6 +529,48 @@ class LookupSenderValuesetsMapper : Mapper {
     }
 }
 
+/**
+ * The lookupSenderAutomationValuesetsMapper is used to lookup values from the
+ *      "sender_automation_value_set_row" table/csv
+ * The args for the mapper are:
```

Comment:
It sounds from here that there are multiple arguments.  Is that true?  Maybe add an example usage like when used in a schema.

Reply:
there is now more than one argument :smile: